src/index.js

```diff
@@ -1,4 +1,5 @@
 /* @flow */
+import "babel-polyfill";
```

Comment:
I wish we could handle this without having to add babel-polyfill back in the mix.